transfers/get-accession-number

```diff
@@ -7,9 +7,9 @@ def main(dirname):
     # Expecting a directory name like sometext---dc.identifier---accession
     parts = dirname.rsplit('---', 1)
     try:
-        print('"' + parts[1] + '"')  # Accession ID must be quoted
+        print('"' + parts[1] + '"', end='')  # Accession ID must be quoted
```

Comment:
Since this is being printed, my preference'd be to print with a newline; it's standard for the output of CLI tools to end with a newline. (POSIX actually [defines](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) a "line" of text as ending with a newline; some commandline tools in pipelines may act strangely if data doesn't end with a newline.